roles/file-management/remove-files/tasks/remove-file.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
please consider naming this file `flag-file-for-removal.yml` or something like that as the actual removal happens in `mainl.yml`